crates/kitsune_p2p/dht_arc/src/test/intersection.rs

```diff
@@ -38,6 +38,7 @@ fn test_intersection_at_limits() {
 }
 
 #[test]
+// @maackle Can we delete this if we don't care about edge case?
 #[ignore = "This test should pass, but doesn't.
 It seems to highlight a weird edge case in the underlying `intervallum` crate,
 but shouldn't have an impact on our use"]
```

Comment:
I like keeping this in. It documents some weird behavior that we don't want, but we can tolerate, and if it does wind up causing problems down the road, we have a clue that someone may stumble upon some day.

We haven't talked about it but I think this is a legit use of ignored tests as living documentation.